wakuv2/waku.go

```diff
@@ -1344,21 +1354,19 @@ func (w *Waku) OnNewEnvelopes(envelope *protocol.Envelope, msgType common.Messag
 
 // addEnvelope adds an envelope to the envelope map, used for sending
 func (w *Waku) addEnvelope(envelope *common.ReceivedMessage) {
-	hash := envelope.Hash()
-
 	w.poolMu.Lock()
-	w.envelopes[hash] = envelope
+	w.envelopes.Set(envelope.Hash(), envelope, ttlcache.DefaultTTL)
```

Comment:
Use `cacheTTL` here, not `ttlcache.DefaultTTL`?

Reply:
According to the docs, `DefaultTTL` indicates that the default TTL value of the cache instance should be used, (in this case the `cacheTTL` value)